### ISSUE TITLE
Solved - Bug: --le=renew returns - cannot renrew because already expired cert

### ISSUE THREAD

**Issue by s-a-s-k-i-a** [EasyEngine/easyengine#849]
When running 
ee site update domain.com --le=renew

on a site where the ssl cert has 0 days left, we can not renew the sll cert. The script returns the following message:
Renewing SSl cert for https://domain.com
ERROR : Cannot RENEW SSL cert !
Your current cert already EXPIRED !
Check logs for reason `tail /var/log/ee/ee.log` & Try Again!!!

I tried using --letsencrypt=off
and rerun
--letsencrypt

But same issue. It returns that there is already a cert installed. Try renew.
Renew doesnt work.

Pls fix asap. I am seeing this issue on various servers and sites.

Thanks.

**Comment by 9553940** [comment]
I had the same issue with some of my domains (not sure why some worked, and some didn't).

I figured out a workaround:

`ee site update domain.com --letsencrypt=off`

Then follow this guide up to "After successful verification, you will receive following message."
(https://easyengine.io/tutorials/nginx/letsencrypt/)

Then finally 
`ee site update domain.com --letsencrypt=on`


Also, if you're using CloudFlare, make sure you turn on Developer mode under Cache if you're using their reverse proxy.

**Comment by s-a-s-k-i-a** [comment]
HM, I did follow your advice, however I am seeing this in console:

 - The following errors were reported by the server:

   Domain: mydomain.de
   Type:   unauthorized
   Detail: Invalid response from
   http://mydomain.de/.well-known/acme-challenge/TPXaGCJ_Ycm-fp6pQkCPqr1kgKx-LQvOHalcoHrENjI:
   "<!DOCTYPE html>
   <html lang="de-DE" prefix="og: http://ogp.me/ns#">
   <head>
   <meta charset="UTF-8">
   
   <m"

   Domain: www.mydomain.de
   Type:   unauthorized
   Detail: Invalid response from
   http://www.mydomain.de/.well-known/acme-challenge/i1gMYnPXSHM-wptXwZ6rBB07q2bKqzFeDL8hLTtpfT8:
   "<!DOCTYPE html>
   <html lang="de-DE" prefix="og: http://ogp.me/ns#">
   <head>
   <meta charset="UTF-8">
   
   <m"

   To fix these errors, please make sure that your domain name was
   entered correctly and the DNS A record(s) for that domain
   contain(s) the right IP address

Both www and non-www variant of the domain point to my server IP. 
This needs some bugfixing, honestly.

**Comment by janiosarmento** [comment]
This is a Let’s Encrypt issue, not EE’s.

Did you complain on their Github too?

**Comment by s-a-s-k-i-a** [comment]
Oh I ddint even know about le's github. Thanks
 for the heads up.
I will link this thread in their issue table

**Comment by s-a-s-k-i-a** [comment]
And another update:
I tried the solution once more on one of the domains:

I had the same issue with some of my domains (not sure why some worked, and some didn't).

I figured out a workaround:

ee site update domain.com --letsencrypt=off

Then follow this guide up to "After successful verification, you will receive following message."
https://easyengine.io/tutorials/nginx/letsencrypt/

Then finally
ee site update domain.com --letsencrypt=on

Also, if you're using CloudFlare, make sure you turn on Developer mode under Cache if you're using their reverse proxy.

And it worked now, even though it didnt this morning. odd.
